### ISSUE TITLE
gcc does not recognize -fmax-errors

### ISSUE THREAD

**Issue by Tiwalun** [hebaishi/linter-gcc#47]
I have got an older version of GCC (gcc version 4.1.2 (Altera 11.1 Build 173)), which does not recognize the -fmax-errors=0 option, which gets added by linter-gcc:

    cc1.exe: error: unrecognized commandline option "-fmax-errors=0"

Would it be possible to add an option to disable this, so that linter-gcc does not add this flag?

**Comment by hebaishi** [comment]
I will add an option to disable ```-fmax-errors```. Thank you for bringing this to my attention.

**Comment by hebaishi** [comment]
The change has been made. To completely switch off ```-fmax-errors```, simply set the number of max errors to ```-1```. Please update linter-gcc to use this feature. Note that a restart of Atom is required to reload the package.

**Comment by hebaishi** [comment]
Hello again. Is everything working as expected?

**Comment by Tiwalun** [comment]
Hi, thank you very much for adding this! It works perfectly :smile:

**Comment by stretchtiberius** [comment]
Could you add a comment in the settings indicating this option? I don't think many people would realize that this is an option. I saw this option as a new feature in the changelog, but had to search here to find how to do it.

Also, I think this feature would be helpful to anyone who is using your linter with clang which works very similarly to gcc, but does not support the ```--fmax-errors``` parameter. Thanks!

**Comment by hebaishi** [comment]
Sure! I should have done that when I made the change. Update on the way!

**Comment by hebaishi** [comment]
Done! Simply update to see the change!

**Comment by stretchtiberius** [comment]
Thanks! That was quick!
